retry.go

```diff
@@ -183,7 +197,7 @@ func (r *realisClient) thriftCallWithRetries(thriftCall auroraThriftCall) (*auro
 			// If there was no client error, but the response is nil, something went wrong.
 			// Ideally, we'll never encounter this but we're placing a safeguard here.
 			if resp == nil {
-				return nil, errors.New("Response from aurora is nil")
+				return nil, errors.New("response from aurora is nil")
```

Comment:
Could define a sentinel error `realis.ErrNilResponse` like `io.EOF`, though consider x/erros is comming in go1.13, you might want to defer error handling stuff until that. (btw: there are still people using go1.9 w/ gorealis)